js/apps/kc-create/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "kc-create",
+  "version": "999.0.0-SNAPSHOT",
+  "description": "Create a new Keycloak ui project based on a template",
+  "main": "create.js",
```

Comment:
I don't believe there is a [`main` ](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main) needed here, as that defines an entry-point that can be imported, but `create.js` is meant to be used as a CLI, it is not an importable entry-point.